src/constants/common.js

```diff
@@ -87,4 +87,27 @@ module.exports = {
 	MEDIUM: 'medium',
 	RECOMMENDED_FOR: 'recommended_for',
 	CATEGORIES: 'categories',
+	jobsToCreate: [
```

Comment:
This basically removes the one time creation of jobs in scheduler service completely right?

Reply:
@joffinjoy Yes.